enterprise/api/v1/views.py

```diff
@@ -313,8 +318,8 @@ def enroll_learners_in_courses(self, request, pk):
                 )
 
             self._create_ecom_orders_for_enrollments(
-                course_run['key'],
-                course_run['mode'],
+                course_run,
+                course_runs[course_run],
```

Comment:
if this actually represents the course_mode, I would just call it that here.. not obvious from this.. but have to read the function def which says 

```
                                            course_run_key,
                                            mode,
                                            discount,
                                            salesforce_id,
                                            succeeded_enrollments
```

not sure if this works but is renaming `course_runs` to `course_modes` going to help? looks like `course_runs` is now different from what it was and only holds course modes?